server/handlers/sources.go

```diff
@@ -79,6 +86,25 @@ func (sh *SourcesHandler) ClearCacheHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, middleware.OKResponse())
 }
 
+func (sh *SourcesHandler) deleteWareHouseData(driver driversbase.Driver, destinationIds []string, sourceID string, collection string, multiErr error) {
```

Comment:
Enriching input error doesn't work properly. You should explicitly return multiErr.

Reply:
Added multiErr return statement